### PR TITLE
Adding the latest plugin/stable PennyLane column

### PR DESCRIPTION
**Pull request by antalszava**
In certain cases, it can be useful to see the status of the latest plugin version against the stable PennyLane version. This PR adds the latest plugin/stable PennyLane column.

Further updated `README` to reflect the changes.

### CODE REVIEW

File: .github/workflows/aqt-latest-stable.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Workflow template for testing plugins against PennyLane stable
```
Comment (antalszava) [inline .github/workflows/aqt-latest-stable.yml:1]: These were generated by using the updated `compile.py`

File: workflow-template-stable.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
-# Workflow template for testing plugin stable/PennyLane stable
+# Workflow template for testing plugins against PennyLane stable
 
-name: {{ plugin }}-stable-stable
+name: {{ plugin }}-{% if latest %}latest{% else %}stable{% endif %}-stable
```
Comment (antalszava) [inline workflow-template-stable.yml:3]: extended with conditional logic

File: .github/workflows/aqt-latest-latest.yml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Workflow template for testing plugins aginst PennyLane latest
+# Workflow template for testing plugins against PennyLane latest
```
Comment (antalszava) [inline .github/workflows/aqt-latest-latest.yml:1]: correction of small typo --- a reason for the added unrelated `yml` files as changed ones

File: README.md [hunk 4]
```diff
@@ -11,17 +11,17 @@ Finally, tests can be triggered manually via the 'Actions' tab.
...
+|           | PyPI version                                                                                   | Stable plugin/stable PennyLane                                                                                                                                                                                                                          | Stable plugin/latest PennyLane                                                                                                                                                                                                                          | Latest plugin/ stable PennyLane                                                                                                                                                                                                                          | Latest plugin/latest PennyLane                                                                                                                                                                                                                          |
```
Comment (antalszava) [inline README.md:14]: The main change here is adding `| Latest plugin/ stable PennyLane` and a new entry for each plugin (perhaps good to look at the rendered version of the file)

File: README.md [hunk 5]
```diff
@@ -51,6 +51,9 @@ All entries in the matrix are tested against PennyLane latest (GitHub master).
   - **Latest/latest failing**: Updates must be made to the plugin to ensure GitHub
     master passes.
 
+  - **Latest/stable failing**: The requirement of PennyLane in the plugin will
+    have to be increased.
```
Comment (josh146) [inline README.md:55]: Nice 💪

**Review by josh146 — APPROVED** [review]
Awesome, thanks for adding this @antalszava! It's a bit hard to read due to the autogenerated workflows, but I looked specifically at the SF and Qiskit cases and they looked good :+1:

**Pull request merged**